include/macros.h

```diff
@@ -129,4 +129,12 @@ extern GraphicsContext* __gfxCtx;
 
 #define VTX_T(x,y,z,s,t,cr,cg,cb,a) { { x, y, z }, 0, { s, t }, { cr, cg, cb, a } }
 
+#ifdef NDEBUG
+#define assert(cond, msg, file, line) ((void) 0)
```

Comment:
All our macros so far are uppercase so this should be too.

Reply:
Done :D